### PR TITLE
Docs/c4

### PR DESCRIPTION
**Pull request by kaniak274**
## Description
This pull request adds C4 diagrams for modular monolith app in Flask and two frontend app in Svelte.

### Core modules
I splitted core of the system into 4 microservices:
* Users - Obviously user for users authentications.
* Cards - Contains data about Pokemon cards.
* Offers - Stores information about offers.
* Transfers - Module to cover transfers information.

### Frontends
I splitted frontends into one for admin to manage cards and auctions and other for sellers and buyers.

### Authentication
For authentication I selected `Cognito` as it is free. Although, we can use something else or write it ourselves if you want to.

### CODE REVIEW

File: docs/structure/PokeCards/Backend/container.puml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Container_Boundary(pokemoncards, "Backend") {
+    Container(cards, "Pokemon Cards", "TBA", "List with pokemon cards managed by admin")
+    Container(cardsdb, "Pokemon Cards DB", "PostgreSQL", "Stores Pokemon cards")
+
+    Container(offers, "Offers", "TBA", "Contains everything related to the auctions")
+    Container(offersdb, "Offers DB", "PostgreSQL", "Stores auctions data")
+
+    Container(users, "Users", "Python, 2-layered", "Users module to connect with Cognito")
+
+    Container(transfers, "Transfers", "TBA", "Transfers module with PayPal integration")
```
Comment (KamilSupera) [inline docs/structure/PokeCards/Backend/container.puml:17]: This TBA here, do you have any options?

Comment (kaniak274) [reply docs/structure/PokeCards/Backend/container.puml:17]: Mostly we can choose from:
* Layered / Onion architecture,
* Hexagonal architecture,

I didn't decided yet + I want to involve you as well 😉. I decided for users as this is the most obvious choice 'cause nothing important is happening there for the process

Comment (kaniak274) [reply docs/structure/PokeCards/Backend/container.puml:17]: Ah we can use one more thing which I doubt we will use here. The [pipes & filters](https://docs.microsoft.com/pl-pl/azure/architecture/patterns/pipes-and-filters) 😁

**Comment by kaniak274** [comment]
I merged this pr 'cause I need it and it's not really that important rn, we can always change it 😁

**Pull request merged**